CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 3.2.0 - 2019-07-31
```

Comment:
Don't worry too much about hte changelog next time, as I will make a release with more than the Hotel Ratings API :) I will manage this during the release